src/components/PokeCard.js

```diff
@@ -0,0 +1,44 @@
+import './PokeCard.css';
+import React, { useState } from 'react';
+import { mockPokemonData } from '../mock/pokeData';
+
+function getData() {
+    return Promise.resolve( mockPokemonData );
+}
+
+const initialPokemonState = {
+    name: '',
+    sprites: {
+        front_default: '',
+        front_shiny: ''
+    },
+    video: ''
+};
+
+function PokeCard() {
+    const [pokemon, setPokemon] = useState(initialPokemonState);
+
+    getData()
+        .then(pokemon => {
+             setPokemon(pokemon)
+        });
+
+    return (
+        <div className="poke-card"> 
+            <div className="pokemon-header">
+                <h1 className="pokemon-name">{pokemon.name}</h1>
+            </div>
+            <div className="pokemon-images">
+                <img src={pokemon.sprites.front_default} alt="default" />
```

Comment:
You could use destructuring to keep the JSX a bit cleaner like in:
```js
const { name, sprites: { front_default: frontDefault, front_shiny: frontShiny } = {}, video } = pokemon;
```
- `frontDefault` and `frontShiny` for renaming the original properties
- then at this line:
```suggestion
                <img src={frontDefault} alt="default" />
```